packages/react-components/src/templates/UserProfileHeader.tsx

```diff
@@ -292,7 +294,12 @@ const UserProfileHeader: React.FC<UserProfileHeaderProps> = ({
           </TabLink>
           {useFlags().isEnabled('EVENTS_SEARCH') && (
             <TabLink href={tabRoutes.upcoming({}).$}>
-              Upcoming Events {` (${upcomingEventsCount})`}
+              Upcoming Events {`(${upcomingEventsCount})`}
+            </TabLink>
+          )}
+          {useFlags().isEnabled('EVENTS_SEARCH') && (
```

Comment:
Does this need to be duplicated? Maybe just use a fragment.

Reply:
I tried fragment and it breaks the layout 😢

Reply:
Ah! Your right, tab nav iterates through the elements so it wont work. Sorry!